src/Testcontainers.CosmosDb/CosmosDbBuilder.cs

```diff
@@ -45,7 +45,7 @@ protected override CosmosDbBuilder Init()
         return base.Init()
             .WithImage(CosmosDbImage)
             .WithPortBinding(CosmosDbPort, true)
-            .WithWaitStrategy(Wait.ForUnixContainer().UntilMessageIsLogged("Started"));
+            .WithWaitStrategy(Wait.ForUnixContainer().UntilMessageIsLogged("Started(?! )"));
```

Comment:
```suggestion
            .WithWaitStrategy(Wait.ForUnixContainer().UntilMessageIsLogged("^Started$"));
```

Thanks for the PR. Wouldn't this regular expression simpler?

Reply:
This won't work unless `RegexOptions.Multiline` is used (it will also need to be amended to consider a carriage return), i.e.

``` csharp
new Regex(@"^Started\r?$", RegexOptions.Multiline)
```

I went with the negative lookahead for simplicity. The regex above should probably also be compiled and have a timeout; happy to refactor to this approach.

Reply:
You are right. I missed the new line in the referenced issue's console output. Then `_ = new Regex("Started\\r\\n$", RegexOptions.None, TimeSpan.FromSeconds(1));` should work right? I do not think we need the compile flag, the regular expression does not process a lot of data.

Reply:
Without the multiline flag, I don't believe this will cater for scenarios in which anything gets logged _after_ the "Started" message. Happy to run with `"Started\\r?\\n"`?

```suggestion
            .WithWaitStrategy(Wait.ForUnixContainer().UntilMessageIsLogged("Started\\r?\\n"));
```

Reply:
Sounds good, let's do it that way.